torch/cuda/amp/common.py

```diff
@@ -7,4 +7,8 @@
 
 
 def amp_definitely_not_available():
-    return not (torch.cuda.is_available() or find_spec("torch_xla"))
+    return not (
```

Comment:
Why do you change this function? I would expect only the one from torch.amp is used and should work fine.

Reply:
I see, I didn't realize this function wasn't called.